todo_app/app.py

```diff
@@ -2,22 +2,29 @@
 import requests
 import os
 import datetime
+from bson.json_util import dumps
+from bson.json_util import loads
 from todo_app.flask_config import Config
 from todo_app.classModels import Item, ViewModel
+import pymongo
 
 def create_app(): 
     app = Flask(__name__)
     app.config.from_object(Config)
-
-    API_KEY = os.environ.get("api_key")
-    TOKEN = os.environ.get("token")
-    TRELLO_BOARD_ID = os.environ.get("TRELLO_BOARD_ID")
+    username=os.getenv("MONGO_USER")
+    password=os.getenv("MONGO_PASSWORD")
+    url=os.getenv("MONGO_URL")
+    database=os.getenv("MONGO_DB")
+    protocol=os.getenv("MONGO_PROTOCOL")
+    collection=os.getenv("MONGO_COLLECTION")
+    MONGO_URI="{0}{1}:{2}@{3}/{4}?retryWrites=true&w=majority".format(protocol,username,password,url,database)
+    mongo = pymongo.MongoClient(MONGO_URI)
+    collections = mongo.myDatabase.list_collection_names()
+    if collection not in collections:
+        todos = mongo.MyDatabase[collection]
```

Comment:
This works, but slightly fortunately!

First is gets the names of collections in "myDatabase" db (note the lowercase m!) which is empty - it then grabs the correct collection from the "MyDatabase" db.
Instead we should always set this todos variable - it doesn't matter if the collection already exists or not
```suggestion
    todos = mongo.MyDatabase[collection]
```